modules/effects/schematics/ng-add/schema.json

```diff
@@ -12,15 +12,20 @@
     "skipPackageJson": {
       "type": "boolean",
       "default": false,
-      "description":
-        "Do not add @ngrx/effects as dependency to package.json (e.g., --skipPackageJson)."
+      "description": "Do not add @ngrx/effects as dependency to package.json (e.g., --skipPackageJson)."
     },
     "path": {
       "type": "string",
       "format": "path",
       "description": "The path to create the effect.",
       "visible": false
     },
+    "project": {
```

Comment:
Nit: Could you move this to above `module`, because it's located there in every other file.